static/src/js/components/OrderStatus/OrderStatusItem/__tests__/DownloadFilesPanel.test.js

```diff
@@ -79,5 +79,23 @@ describe('DownloadFilesPanel', () => {
       expect(windowActions.props().disableCopy).toEqual(true)
       expect(windowActions.props().disableSave).toEqual(true)
     })
+    test('hides the edd button', () => {
+      const enzymeWrapper = shallow(
+        <DownloadFilesPanel
+          accessMethodType="ESI"
```

Comment:
Very minor, but I think this makes more sense if it is "download". 

And remove `retrievalCollection={{}}` below, it looks like that prop isn't even used for the component anymore